src/App.js

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, NavLink, Route } from 'react-router-dom';
+import List from './components/List.js';
+import AddItem from './components/AddItem.js';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<List />} />
+          <Route path="/additem" element={<AddItem />} />
+        </Routes>
+
+        <div className="link-div">
+          <NavLink
+            to="/"
+            style={({ isActive }) =>
+              isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
```

Comment:
Thinking about your comment, I believe that this can be removed and we can instead have a CSS `.active` selector, i.e.: 

```
.active {
  font-weight: bold;
}
```

Reply:
@rachael-t Thanks for the feedeback! My original comment was actually regarding the commit and review request from @lena-A-Al from a few days ago before I added the conditional styling. We began with trying to change styling on click, which ultimately wasn't going to work, so my comment was referring to that. Sorry- I'm still trying to figure out the PR work flow! I'm afraid I just made the convo confusing by retroactively responding to an out-of-date commit and review request!

As for your suggestion, I think we would still have to conditionally apply the class to the component depending on the current path, so I'm not sure we can fully ditch the conditional rendering. Unless I'm misunderstanding something? I ran across a solution that conditionally applies an .active class and styles according to that, but I chose to use the style attribute because it seemed more explicit. I think either way is valid though! Maybe @emilygracekz can chime in?

Reply:
Oh, that makes sense! Sorry about that on my end, and no worries, as we are all figuring it out together 😄 I think we can leave it as it is, as it is working great! When we move to mentor review they can chime in and add their two cents.

Reply:
heya @rachael-t and @nikkigraybeal! 

I think this is a great convo about how to apply the styling, and Nikki I think you make a good case about using the `style` attribute. 

This decision comes down to preference. I usually go with Rachels method of conditionally applying a `active` `className`.